frontend/src/pages/home/learn.tsx

```diff
@@ -4,13 +4,61 @@ import { GetServerSideProps } from 'next'
 import { useTranslation } from 'next-i18next'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
 import { NextSeo } from 'next-seo'
+import {
+  Card,
+  CardHeader,
+  CardBody,
+  Typography,
+} from "@material-tailwind/react";
 
 const Learn: FC = () => {
   const { t } = useTranslation('learn')
+  const sections = t('sections', { returnObjects: true });
```

Comment:
I discourage the use of `returnObjects` options.

Reasons:
- when using this option, you are required to use camel case to map the content to the JavaScript object naming convention, even though this project uses kebab case in the JSON files. Resulting in different naming conventions in JSON files.
- when using this option in Typescript, the return type of `t` will be set to `any`. This can lead to difficulties in debugging because it may be hard to determine whether there is an error in the key name in the JSON file or in the JavaScript code. TypeScript cannot validate the `myKey` in the JSON, so it may be used incorrectly with `{myKeys}` in React, resulting in nothing being displayed, which may go unnoticed. This issue can also occur when dealing with child keys, as errors may only be caught at runtime. For example, if the JSON has `"overview": { "header": "My Header" }`, a typo in the usage of `{overviews.header}` would result in `overviews` being `undefined` and `header` being inaccessible. Using `t` or `Trans` component with the `i18nKey` is a better option for debugging and reducing errors, as any key errors in the JSON or JavaScript will be outputted as `common:overviews.header`, for example.

Reply:
Updated this, errors will now display in the components so that they are identifiable.